include/pingcap/kv/internal/type_traits.h

```diff
@@ -14,6 +14,7 @@ struct RpcTypeTraits
 {
 };
 
+// Note that this macro do not applicable for grpc streaming call
```

Comment:
```suggestion
// Note that this macro only not applicable for grpc unary call
```

Reply:
"Note that this macro is only applicable for grpc unary call" maybe better?